crates/fluvio/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fluvio"
-version = "0.11.0"
+version = "0.12.0"
```

Comment:
I think that a patch update is ok for this change, (0.11.1)

Reply:
After talking with Nick C, the changes to the ConsumerConfig were undone, making a version change unnecessary as no changes are made to fluvio crate now.